stories/avatar-group/avatar-group.stories.js

```diff
@@ -0,0 +1,435 @@
+export default {
+    title: 'Components/Avatar Group',
+    parameters: {
+        description: `The avatar group displays a number of avatars, which act as a digital representation of a user. This control is useful for visualizing a team, project team members, or a group of employees, for example.
+
+
+
+## Sizes
+
+The control allows you to display the avatars in different sizes, depending on your use case.
+
+| **Size** | &nbsp;&nbsp; **Group type** | &nbsp;&nbsp; **Individual type** | **Modifier class** |
+| :--------- | ----------: | :----------------------- | ---------------: |
+| XS | &nbsp;&nbsp; -0.75rem | &nbsp;&nbsp; 0.0625rem | <code>--xs</code> |
+| S | &nbsp;&nbsp; -1.25rem | &nbsp;&nbsp; 0.125rem | <code>--s</code> |
+| M | &nbsp;&nbsp; -1.625rem | &nbsp;&nbsp; 0.125rem | <code>--m</code> |
+| L | &nbsp;&nbsp; -2rem | &nbsp;&nbsp; 0.125rem | <code>--l</code> |
+| XL | &nbsp;&nbsp; -2.75rem | &nbsp;&nbsp; 0.25rem | <code>--xl</code> |
+
+##Types
+
+The **AvatarGroup** control has two group types:
+
+- **Group type**: The avatars are displayed as partially overlapped on top of each other and the entire group has one click/tap area.
+- **Individual type**: The avatars are displayed side-by-side and each avatar has its own click/tap area.
+
+## Usage
+
+Use the **AvatarGroup** if:
+
+- You want to display a group of avatars.
+- You want to display several avatars which have something in common.
+
+Do not use the **AvatarGroup** if:
+
+- You want to display a single avatar.
+- You want to display a gallery for simple images.
+- You want to use it for other visual content than avatars.
+
+`,
+        docs: { iframeHeight: 250 },
+        tags: ['a11y', 'f3', 'theme'],
+        components: ['avatar-group', 'avatar', 'icon', 'button', 'popover', 'bar', 'layout-grid', 'title', 'quick-view', 'title', 'link', 'form-layout-grid', 'form-group', 'form-item', 'form-label', 'input']
+    }
+};
+
+export const individualType = () => `<div class="fd-avatar-group fd-avatar-group--individual-type fd-avatar-group--s">
+    <div class="fd-popover">
+        <div class="fd-popover__control">
+            <div class="fd-avatar-group__item" tabindex="0" aria-haspopup="true" aria-expanded="false" onclick="onPopoverClick('popover_avatar_5z28edb');">
+                <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--border" aria-label="Wendy Wallace">WW</span>
```

Comment:
This appears as a grey border in Quartz mode and I believe it should not be there at all.  HCW/HCB look good though

Reply:
This element has `fd-avatar--border` which adds a `border: 0.0625rem solid --sapGroup_ContentBorderColor`
<img width="1070" alt="Screen Shot 2021-01-22 at 4 16 25 PM" src="https://user-images.githubusercontent.com/39598672/105548045-314b4680-5ccd-11eb-8f79-c6b9d7e7ac26.png">